setup.py

```diff
@@ -5,7 +5,7 @@
 import sys
 from setuptools import setup
 
-version = '1.1.0'
+version = '2.0.0'
```

Comment:
Bumping the major version as moving `pyjwt` to an optional dependency is a breaking change.